Camera/examples/example-geojsonfeature.json

```diff
@@ -27,6 +27,7 @@
       ]
```

Comment:
this file is generated automatically. You do not need it on the PR (we try you to work the less the better)